warehouse/oidc/views.py

```diff
@@ -39,6 +39,7 @@ class JsonResponse(TypedDict, total=False):
     message: str | None
     errors: list[Error] | None
     token: StrictStr | None
+    expires: int | None
```

Comment:
@miketheman something like this?

```suggestion
    # This value is used to communicate the minted token's expiration
    # time to the user. It is not used to determine the expiration,
    # changing this field does not change the token's expiration time.
    expires: int | None
```

Reply:
I don't have permissions to push to this branch, I'll leave the suggestion here so that @woodruffw can accept it.

Reply:
Done, thanks @facutuesca 🙂